packages/shared/src/index.ts

```diff
@@ -10,3 +10,5 @@ export * from './ShadowRoot'
 export * from './TypedMessage'
 export * from './utils'
 export * from './hooks'
+export * from './database'
```

Comment:
Well database is the inner implementation detail of the Mask. It should not be shared.

Reply:
Currently just moving the type to shared.